dockstore-webservice/src/main/java/io/dockstore/webservice/core/Version.java

```diff
@@ -217,6 +217,10 @@
     @BatchSize(size = 25)
     private Set<Image> images = new HashSet<>();
 
+    @Column(columnDefinition = "TEXT")
+    @ApiModelProperty(value = "The language version for the version's descriptor files. If there are multiple versions, then this contains the highest version")
```

Comment:
Apologies, this will need to contain all the versions as an array. Not so much for a (correct) WDL workflow but for CWL workflows.  https://github.com/ga4gh/tool-registry-service-schemas/blob/v2.0.1/openapi/openapi.yaml#L632-L647

Contrary to my other comment, if we're storing this anyway, spin out a separate ticket since throwing up a validation warning message for multiple WDL versions would be trivial anyway.